model/src/DA/Finance/Asset/Settlement.daml

```diff
@@ -63,6 +63,7 @@ template AssetSettlementRule
         do
           deposit <- fetchAndArchive depositCid
           deposit.account === account
+          deposit.lock === None
```

Comment:
Do I understand this correctly, that the current AssetSettlementRule completely ignores the new locking mechanism? So the idea would be to remove this at the same time we merge in the new AccountRule?

Reply:
Yes, this is correct on both questions. The reasons for this are :

1. To create a clean solution without having to deal with the limitations of the current model;
2. Renaming of the template and file - I find the current naming doesn't reflect how it's actually used and can lead to confusion. Of course, this is subjective so let me know if you don't agree.